projects/1.20-fabric/assets/chatshot/chatshot/lang/zh_cn.json

```
@@ -0,0 +1,27 @@
+{
+  "chatshot.copy": "§e§l复制到剪贴板",
+  "chatshot.clickText": "§b§l点击 §r§8- §7复制为文本",
+  "chatshot.clickImage": "§b§l点击 §r§8- §7复制为图片",
+  "chatshot.shiftClickText": "§b§l按下Shift键并点击 §r§8- §7复制为文本",
+  "chatshot.shiftClickImage": "§b§l按下Shift键并点击 §r§8- §7复制为图片",
+  "chatshot.text.success": "§7聊天消息文本已经复制到剪贴板。",
+  "chatshot.image.success": "§7聊天消息截图已经复制到剪贴板。",
+  "chatshot.image.fail": "§7无法将聊天消息截图复制到剪贴板。",
+  "chatshot.config": "ChatShot 配置",
+  "chatshot.shadow": "文本阴影",
+  "chatshot.shadow.description": "聊天消息截图中文本是否有阴影。",
+  "chatshot.scale": "文本缩放",
+  "chatshot.scale.description": "聊天消息截图中文本的缩放比例。",
+  "chatshot.tooltip": "显示工具提示",

```

```suggestion
  "chatshot.tooltip": "显示提示框",
```
https://zh.minecraft.wiki/w/%E6%8F%90%E7%A4%BA%E6%A1%86
下同